### ISSUE TITLE
Evaluation of the model

### ISSUE THREAD

**Issue by leochli** [twoflypig/DrQA#1]
Hi,
It seems that you're training and testing with the Sougou QA set.
Have you evaluated the performance of your model in SQuAD dataset？ If so, what is the resulting EM and F1 score?

**Comment by twoflypig** [comment]
@leochli Hi,thanks for your questions! I did join the Sougou QA competition last December .  The results that I last submitted are as follows:

method|test set  | validate set
---|---- | ---
F1| 0.4350 |0.4580
Accuracy| 0.3544  | 0.3822 
Final score|0.3947  | 0.4201

The performance was really bad compared with other  teams' . I think it's better to use the code written in  Pytorch  released by  Facebook .Finally , I truely appreciate for your questions ! 
Best wishes.
